tests/cds-rdm/test_full_migration.py

```diff
@@ -151,6 +155,113 @@ def multiple_versions_with_cs(record):
+    assert "custom_fields" in dict_rec
+    assert dict_rec["custom_fields"] == {
+        "cern:accelerators": [
+            {"id": "CERN AD", "title": {"en": "CERN AD"}},
+            {"id": "CERN AD", "title": {"en": "CERN AD"}},
+        ],
+    }
+
+
+def file_missing(record):
+    """2783103"""
+    dict_rec = record.to_dict()
+    assert "files" in dict_rec
+    assert dict_rec["files"]["enabled"] == False
+    assert dict_rec["files"]["count"] == 0
+    assert "media_files" in dict_rec
+    assert dict_rec["media_files"]["enabled"] == False
+    assert dict_rec["media_files"]["count"] == 0
+
+
+def file_restricted(record):
+    """2783104"""
+    dict_rec = record.to_dict()
+    assert "access" in dict_rec
+    assert dict_rec["access"]["record"] == "public"
+    assert dict_rec["access"]["files"] == "restricted"
+    assert "files" in dict_rec
+    assert dict_rec["files"]["enabled"] == True
+    assert dict_rec["files"]["count"] == 1
+
+
+def irregular_exp_field(record):
+    """2046076"""
+    dict_rec = record.to_dict()
+    assert "custom_fields" in dict_rec
+    assert dict_rec["custom_fields"] == {
+        "cern:accelerators": [
+            {"id": "CERN AD", "title": {"en": "CERN AD"}},
+        ],
+    }
+
+
+def custom_affiliation(record):
+    """2041388"""
+    dict_rec = record.to_dict()
+    assert "affiliations" in dict_rec["metadata"]["creators"][0]
+    assert dict_rec["metadata"]["creators"][0]["affiliations"] == [
+        {"name": "Department of Physics Royal Institute of Technology Stockholm Sweden"}
+    ]
+
+
+def contains_aleph(record):
+    """1597985"""
+    dict_rec = record.to_dict()
+    assert "identifiers" in dict_rec["metadata"]
+    assert dict_rec["metadata"]["identifiers"] == [
+        {"identifier": "CERN-STUDENTS-Note-2013-181", "scheme": "cds_ref"},
+        {"identifier": "000733613CER", "scheme": "aleph"},
+    ]
+
+
+def contains_keywords(record):
+    """2779426"""
+    dict_rec = record.to_dict()
+    assert "subjects" in dict_rec["metadata"]
+    assert dict_rec["metadata"]["subjects"] == [
+        {
+            "id": "Detectors and Experimental Techniques",
+            "subject": "Detectors and Experimental Techniques",
+            "scheme": "CERN",
+        },
+        {
+            "subject": "Gaseous detectors, Drift tubes, CSC, MSGC, Microdot chambers, Micromegas, Micropattern, GEM, TPC, RPC, TGC, RICH, electron avalanche, Garfield++, Magboltz"
+        },
+    ]
+
+
+def author_with_inspire(record):
+    """2294138"""
+    dict_rec = record.to_dict()
+    assert "contributors" in dict_rec["metadata"]
+    assert dict_rec["metadata"]["contributors"] == [
+        {
+            "person_or_org": {
+                "type": "personal",
+                "name": "Glatzer, Julian",
+                "given_name": "Julian",
+                "family_name": "Glatzer",
+                "identifiers": [
+                    {"identifier": "00013837", "scheme": "inspire"},
```

Comment:
```suggestion
                    {"identifier": "INSPIRE-00013837", "scheme": "inspire"},
```

Reply:
could you add it to the code? Happy to know the tests fail on changes :)